firecrown-parameter-inference/cosmosis/srd_CS_values.ini

```diff
@@ -0,0 +1,39 @@
+; Parameters and data in CosmoSIS are organized into sections
+; so we can easily see what they mean.
+; There is only one section in this case, called cosmological_parameters
+
+[cosmological_parameters]
+omega_c = 0.05 0.22 0.3
+omega_b = 0.01 0.0448 0.2
+omega_k = 0.0
+n_s =  0.84 0.963 1.1
+A_s = 0.5e-9 2.16e-09 5.0e-9
+h0 = 0.64 0.71 0.82
+w = -1.0
+wa = 0.0
+mnu = 0.0
+
+;[halo_model_parameters]
+;logt_agn = 7.3 7.8 8.3
+; logt_agn = 7.8
+
+[firecrown_two_point]
+ia_bias = -5.0 0.0 +5.0
+alphaz = -5.0 0.0 +5.0
+; ia_bias = 0.0
+; alphaz = 0.0
+alphag =  -1
```

Comment:
Uh, that shouldn't be there, right? Is this config used anywhere?

Reply:
Ah right. Luckily I had just forgotten to update three local files and it is correct in my local file (which is what got used). I just pushed the changes now.

Reply:
Thanks for catching that!